index.html

```diff
@@ -0,0 +1,237 @@
+        <p class="paragraph1">
+          Full-stack Web Development courses offered through Coursera equip learners with knowledge in building web, hybrid, and mobile solutions; understanding front-end frameworks such as Bootstrap 4 and Angular; implementing NoSQL databases using MongoDB; creating the best user
+           experience regardless of device; the basics of ...
+        </p>
+        <h2 class="date">2023.1.15(THU) ~ 16(FRI)</h2>
+        <p class="address">
+          @ International programming school 
+        </p>
+      </section>
+
+      <!-- main-program -->
+      <section id="program" class="main-program">
+        <h2 class="program-title">Main Program</h2>
+        <hr class="line1" />
+        <div class="program">
+          <div class="program-tab">
+            <img src="./img/program_icon_01.png" alt="icon" />
+            <h3 class="program-text1">Lecture</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_02.png" alt="icon" />
+            <h3 class="program-text1">CC Exhibition</h3>
+            <p class="program-text2">
+              Lorem, ipsum amet consectetur adipisicing elit.
+              Consectetur harum velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_03.png" alt="icon" />
+            <h3 class="program-text1">Forum</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_04.png" alt="icon" />
+            <h3 class="program-text1">Workshop</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum velit.
+            </p>
+          </div>
+          <div class="program-tab">
+            <img src="./img/program_icon_05.png" alt="icon" />
+            <h3 class="program-text1">CC Ignite</h3>
+            <p class="program-text2">
+              Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+              Consectetur harum.
+            </p>
+          </div>
+          <a href="#" class="join-btn">Join CC Global Submit 2015</a>
+        </div>
+        <a href="#" class="program-link">SEE THE WHOLE PROGRAM</a>
+      </section>
+
+      <!-- Speakers -->
+      <section class="speakers">
+        <h2>featured Speakers</h2>
+        <hr class="line1" />
+        <div class="speakersContainer">
+        </div>
+        <button class="btn more">
+          <a href="#">MORE</a>
+          <img src="./img/arrow_down.png" alt="arrow icon" />
+        </button>
+        <a href="#" class="btn less">LESS</a>
```

Comment:
- [ ] Your featured speakers section is nice and very similar to the template, but on the desktop version all the speakers should appear and the button 'more' and 'less' should only appear on the mobile version. Kindly make some changes to fix that.

Reply:
done